sys/fuchsia/vcpu.txt

```diff
@@ -0,0 +1,12 @@
+# Copyright 2018 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <zircon/syscalls.h>
+
+zx_vcpu_create(guest zx_handle, options int32, entry intptr, out ptr[out, zx_handle])
```

Comment:
make sure you remove the duplicated calls from hypervisor_guests.txt

Reply:
they should be removed, hypervisor_guests.txt is deleted.